challenge_9/python/shaowen/unit_test.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
```

Comment:
sorry, I just noticed the yearofprogramming home page is updated, it seems there are some new rules out there(especially the testing part)...i will read through it and make new commits later.